collect_app/src/main/res/layout/main_menu.xml

```diff
@@ -89,13 +89,22 @@
                     android:layout_marginTop="@dimen/margin_extra_small"
                     tools:text="@string/manage_files" />
 
+                <TextView
+                    android:id="@+id/app_name"
+                    style="@style/TextAppearance.Collect.Body1"
```

Comment:
This should be `?textAppearanceBody1` so we're referencing the theme attribute rather than our style explicitly. Changing the version sha to do the same thing would be good as well.